.github/workflows/deploy_admin_app.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 jobs:
   DeployAdminApp:
-    name: Deploying Admin App ${{github.ref_name}}
+    name: Deploying Admin App to ${{ (github.ref_name == 'main' && 'testing') || github.ref_name }}
```

Comment:
But of a headscratcher. But this [helped](https://docs.github.com/en/actions/learn-github-actions/expressions#example).